hoot-cmd/src/main/cpp/hoot/cmd/HelpCmd.cpp

```diff
@@ -5,7 +5,7 @@
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
- *
+ * 
```

Comment:
Yea, this came up when I discovered on February 1 that the LicenseTemplate.txt file had a space at the end of a comment line. Seth and I talked about this deciding that since the template file has been like this for quite some time that we would leave it as is. I will do some testing and see what happens with making a change to the template file.